shell/platform/darwin/ios/ios_context.mm

```diff
@@ -24,11 +26,15 @@
     const std::shared_ptr<const fml::SyncSwitch>& is_gpu_disabled_sync_switch) {
   switch (api) {
     case IOSRenderingAPI::kSoftware:
-      FML_CHECK(backend != IOSRenderingBackend::kImpeller)
-          << "Software rendering is incompatible with Impeller.\n"
-             "Software rendering may have been automatically selected when running on a simulator "
-             "in an environment that does not support Metal. Enabling GPU pass through in your "
-             "environment may fix this. If that is not possible, then disable Impeller.";
+      if (backend == IOSRenderingBackend::kImpeller) {
+        FML_LOG(IMPORTANT)
```

Comment:
I just used the existing warning message, with the part about disablling impeller removed :)